GetIntoTeachingApiTests/Services/StoreTests.cs

```diff
@@ -472,11 +491,24 @@ public async void GetTeachingEventAsync_WithId_ReturnsMatchingEvent()
         [Fact]
         public async void GetTeachingEventAsync_WithReadableId_ReturnsMatchingEvent()
         {
+            await SeedMockTeachingEventBuildingsAsync();
             var events = await SeedMockTeachingEventsAsync();
             var result = await _store.GetTeachingEventAsync(events.First().ReadableId);
 
             result.ReadableId.Should().Be(events.First().ReadableId);
             result.Building.Should().NotBeNull();
+        }
+
+        [Fact]
+        public async Task GetTeachingEventBuildings_ReturnsAll()
+        {
+            await SeedMockTeachingEventBuildingsAsync();
+            var events = await SeedMockTeachingEventsAsync();
+            _mockCrm.Setup(m => m.GetTeachingEvents(It.IsAny<DateTime>())).Returns(events);

```

Comment:
Not following this test, why are we mocking `GetTeachingEvents` when we're querying the buildings?